content/luau/concepts/tables/terms/isfrozen/isfrozen.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'isfrozen()'
+Description: 'Returns a boolean based on whether the table is in read-only mode or not.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Booleans'
+  - 'Tables'
+CatalogContent:
+  - 'paths/computer-science'
+---
+
+The **`isfrozen()`** function in the Luau programming language returns a boolean based on whether the table is in read-only mode or not. When a table is in read-only mode, it means that the contents can't be modified, as a measure of data protection.
+
+## Syntax
+
+```pseudo
+isfrozen(table);
+```
+
+The 'isfrozen()' function returns a boolean value, true if the table is in read-only mode and false if it's not.
+
+## Example
+
+```luau
```

Comment:
```suggestion
```lua
```

Reply:
I don't believe luau is supported by markdown code blocks, but Lua can be used to show the syntax highlighting